src/nvim/api/ui.c

```diff
@@ -24,21 +23,13 @@ typedef struct {
 static PMap(uint64_t) *connected_uis = NULL;
 
 void remote_ui_init(void)
+    FUNC_ATTR_NOEXPORT
 {
   connected_uis = pmap_new(uint64_t)();
-  // Add handler for "attach_ui"
-  String method = cstr_as_string("ui_attach");
-  MsgpackRpcRequestHandler handler = {.fn = remote_ui_attach, .async = false};
-  msgpack_rpc_add_method_handler(method, handler);
-  method = cstr_as_string("ui_detach");
-  handler.fn = remote_ui_detach;
-  msgpack_rpc_add_method_handler(method, handler);
-  method = cstr_as_string("ui_try_resize");
-  handler.fn = remote_ui_try_resize;
-  msgpack_rpc_add_method_handler(method, handler);
 }
 
 void remote_ui_disconnect(uint64_t channel_id)
+    FUNC_ATTR_NOEXPORT
```

Comment:
@justinmk Why `FUNC_ATTR_*`? All other `FUNC_ATTR_*` macros are hints for the compiler. I would actually suggest `API_FUNC void api_func(…) API_ASYNC`, but even if you think placing this in the attribute position is better it should not start with `FUNC_ATTR_` (as well as `FUNC_ATTR_ASYNC`).